config/config.go

```diff
@@ -120,7 +120,8 @@ const (
 		"enabled": false
 	},
 	"alerts_email": "", 
-	"service_heartbeat_type": "http"
+	"service_heartbeat_type": "http",
+    "token_expiry_in_seconds": 20
```

Comment:
Lets make this in minutes and keep the same default as in the readme file. Also the default value can be 24 hrs

Reply:
fixed